src/fklearn/training/imputation.py

```diff
@@ -32,23 +33,41 @@ def imputer(df: pd.DataFrame,
         - If "mean", then replace missing values using the mean along the axis.
         - If "median", then replace missing values using the median along the axis.
         - If "most_frequent", then replace missing using the most frequent value along the axis.
+
+    fill_value : Any, (default=None)
+        if not None, use this as default value when some feature only contains NA values.
     """
 
+    columns_to_fill = list()
+    columns_imputable = columns_to_impute
+    if fill_value is not None:
+        df_is_nan = df[columns_to_impute].isna().all(axis=0)
+        columns_to_fill = list(df_is_nan[df_is_nan].index)
+        columns_imputable = list(filter(lambda column: column not in columns_to_fill, columns_to_impute))
```

Comment:
please , avoid declaring a variable and mutating state. one alternative is  to use python's ternary operator like `a = 1 if False else 2`. one way to achieve this is extracting this in another method (python downsides on being real bad to make private methods; so maybe it can be an "auxiliary" method inside this one)

Reply:
I think that keeping the default value outside of the `if` block is clearer, but I'll refactor to an `else` block